src/routes/clients.ts

```diff
@@ -4,9 +4,9 @@ import controller from '../controller/clients';
 const router = express.Router();
 
 router.get('/clients', controller.getClients);
```

Comment:
```suggestion
router.get('/client', controller.getClients);
```

referenced by https://github.com/tm-pe2/ad-api/pull/13/files#r829762041